### DISCUSSION TITLE
Option to process all files in the download folder.

### DISCUSSION THREAD

**Discussion by nydave69** [pymedusa/Medusa#9112] [Ideas & Feature requests]
I find downloads that are needed all the time, I start the download and then change the label to send it to the correct download folder when finished.  This is great for all my download managers except Medusa..  It gives an error that it wasn't snatched. I then have to spend time trying to get it processed.  Only way I found is to use File-bot to name the episode correctly, manually move them to the destination folder, and have Medusa scan the folder. Not Ideal...

Cant tell Medusa that it's snatched (not an option), better would be that it just processes it if it can use it.

**Comment by medariox** [comment]
It already processes all the files in your post-processing folder. What it doesn't do is processing files that already have a wanted quality, in that case you can force the processing in manual pp. If a file is not named properly it can't be processed until the name is adjusted.

**Comment by nydave69** [comment]
Well it doesn't seem to work that way for me,   Just down loaded the first 3 episodes of WonderVision.  Medusa scanned the files and errored that they were not snatched.. next time I do one I'll posed the actual errors. WonderVision had no episodes downloaded since it was added just the other day.  All of the episodes show as wanted.

**Reply by medariox** [reply]
Maybe you are running different software? Not sure. Without debugs logs we can't help anyway.

**Reply by nydave69** [reply]
Here is the info from help....


Medusa Info: | Branch: masterCommit: 2b874c2c6a3c33d653eec6e50598a194627a6f76Version: 0.5.4Database: 44.17
-- | --
Python Version: | 3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 22:22:05) [MSC v.1916 64 bit (AMD64)]
SSL Version: | OpenSSL 1.1.0j 20 Nov 2018
OS: | Windows-7-6.1.7601-SP1
Locale: | en_US.cp1252
  |  
  |  
User: | Plex
Program Folder: | C:\Medusa\Medusa
Config File: | C:\Medusa\Data\config.ini
Database File: | C:\Medusa\Data\main.db
Cache Folder: | C:\Medusa\Data\cache
Log Folder: | C:\Medusa\Data\Logs
Arguments: | --nolaunch --datadir=C:\Medusa\Data
  |

**Comment by p0psicles** [comment]
If you want support create an issue, with the template filled out and debug logs of a forced or manual search of an ep

**Reply by nydave69** [reply]
I thought the error was intentional. That's why I figured an option to disable it and process it anyway..

Ok will submit it as an issue.....
